docs/products/clickhouse/howto/data-service-integration.md

```diff
@@ -12,220 +11,265 @@ import TabItem from '@theme/TabItem';
 
 Connect your Aiven for ClickHouse® service with another Aiven-managed service or external data source to make your data available in the Aiven for ClickHouse service.
 
-:::tip
-Read about data service integration types and methods available with Aiven for
-ClickHouse® in
-[Aiven for ClickHouse® data service integration](/docs/products/clickhouse/concepts/data-integration-overview).
-:::
-
 ## Prerequisites
 
 - You are familiar with the limitations listed in
   [About Aiven for ClickHouse® data service integration](/docs/products/clickhouse/concepts/data-integration-overview#supported-data-source-types).
 - You have an organization, a project, and an Aiven for ClickHouse service in Aiven.
 - You have access to the [Aiven Console](https://console.aiven.io/).
 
-## Create data service integrations {#create-data-service-integration}
+## Create Apache Kafka integrations
+
+:::tip
+Learn about [managed databases integrations](/docs/products/clickhouse/concepts/data-integration-overview#managed-databases-integration).
+:::
+
+Make Apache Kafka data available in Aiven for ClickHouse using the Kafka engine:
 
 1. Log in to the [Aiven Console](https://console.aiven.io/), and go to an organization
    and a project.
 1. From <ConsoleLabel name="services"/>, select an Aiven for ClickHouse service to integrate
    with a data source.
-1. In the service <ConsoleLabel name="overview"/>, go to **Data pipeline**.
-
-   - To create the first integration for that service, click **Add data source**.
-   - To add an integration for that service, click <ConsoleIcon name="plus"/>.
-
-   As a result, the **Data service integrations** wizard opens, showing
-   a list of all data sources available for integration. If there are no data sources to
-   integrate with, create an Aiven-managed service or an integration endpoint by
-   clicking **Create service** or **Add external endpoint** respectively.
+1. On the service's <ConsoleLabel name="overview"/> page, click
+   <ConsoleLabel name="integrations"/> in the sidebar.
+1. On the **Integrations** page, go to the **Data sources** section and click
+   **Apache Kafka Databases**.
```

Comment:
I understand that this comes from the Console UI, but "Apache Kafka Databases" might be potentially misleading since Kafka is not a database. A clearer alternative could be "Apache Kafka Data Sources" or "Apache Kafka Streams", which better reflect Kafka’s role in streaming data.

Reply:
Indeed, this is a technical term used in the console. This term refers to the type of ClickHouse integration, and the database concept here reflects an entity created in ClickHouse to make data coming from Kafka available.

Reply:
@harshini-rangaswamy , I've updated the terminology accordingly :-)